src/app/skills-page/skills-page.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component } from '@angular/core';
+import { FaIconLibrary } from '@fortawesome/angular-fontawesome';
+import { IconPrefix } from '@fortawesome/fontawesome-svg-core';
+import { faAngular, faCss3Alt, faHtml5, faJs, faFigma, faGitAlt, faBootstrap, IconName } from '@fortawesome/free-brands-svg-icons';
+import { faCode, faDatabase, faLanguage, faLaptopCode, faPeopleGroup } from '@fortawesome/free-solid-svg-icons';
+
+
+const SkillsList = [
+  { name: 'Angular', sup: ' ', rate: 2.0, icon: 'angular' as IconName, type: 'fab' as IconPrefix },
```

Comment:
Fajnie, że jest to poza klasą, ale tutaj zamiast tych `as IconName` i `as IconPrefix` powinno być `const SkillsList: Skill[]` i w samym interface `Skill` typ.